libcnb-test/src/lib.rs

```diff
@@ -151,8 +152,14 @@ impl IntegrationTest {
         let temp_app_dir =
             app::copy_app(&app_dir).expect("Could not copy app to temporary location");
 
-        let temp_crate_buildpack_dir = build::package_crate_buildpack(&self.target_triple)
-            .expect("Could not package current crate as buildpack");
+        let temp_crate_buildpack_dir = if self.buildpacks.contains(&BuildpackReference::Crate) {
```

Comment:
Good catch to skip the build :-) (I presume found via the integration tests added to `libcnb-test` which isn't a valid  buildpack, but still)